code/game/objects/items/weapons/lighters.dm

```diff
@@ -302,3 +312,31 @@
 /obj/item/match/firebrand/New()
 	..()
 	matchignite()
+
+
+/obj/item/match/unathi
+	name = "small blaze"
+	desc = "A little flame of your own, currently located dangerously in your mouth."
+	icon_state = "match_unathi"
+	force = 0
+	flags = DROPDEL
+	origin_tech = null
+	w_class = WEIGHT_CLASS_BULKY //to prevent it going to pockets
+
+/obj/item/match/unathi/New()
+	..()
+	matchignite()
+	name = "small blaze"
+	desc = "A little flame of your own, currently located dangerously in your mouth."
+	icon_state = "match_unathi"
+	force = 0
+	attack_verb = null
```

Comment:
This back-and-forth is a little silly, it'd probably be better to override the `matchignite` proc to not change the variables in the first place